test/Text/Shakespeare/I18NSpec.hs

```diff
@@ -1,16 +1,26 @@
 {-# LANGUAGE MultiParamTypeClasses #-}
 {-# LANGUAGE OverloadedStrings     #-}
 {-# LANGUAGE TemplateHaskell       #-}
+{-# LANGUAGE TypeFamilies          #-}
+
 module Text.Shakespeare.I18NSpec
     ( spec
     ) where
 
 import           Data.Text             (Text)
 import           Text.Shakespeare.I18N
+import           Language.Haskell.TH.Syntax (Lift(..)) 
 
 spec :: Monad m => m ()
 spec = return ()
 
-data Test = Test
+class Lift master => YesodSubApp master where
+   data YesodSubAppData master :: *
+
+newtype SubApp master = SubApp
+  {
+     getOrdering :: Ordering
+  }
+
+mkMessage "(YesodSubApp master) => SubApp master" "test-messages" "en" 
 
-mkMessage "Test" "test-messages" "en"
```

Comment:
Well done for getting this working. I'm a little wary of losing this particular test case though, because we must ensure that this change doesn't break anyone else's project.

Could you make it so that we have both of these two test cases?

1. `mkMessage "Test" "test-messages" "en"`
2. `mkMessage "(YesodSubApp master) => SubApp master" "test-messages" "en"`

Reply:
I think, I can.

Reply:
Done.
See:
[https://github.com/Oleksandr-Zhabenko/shakespeare/actions/runs/10902540480](https://github.com/Oleksandr-Zhabenko/shakespeare/actions/runs/10902540480)